### PR TITLE
Action header

### PR DESCRIPTION
**Pull request by BenLanglois**
Header files for Actions, which include Moves and Items. Still working on the implementation files on the action-implementation branch.

### CODE REVIEW

File: game/debuffHeal.h [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+
+#include <string>
+#include <vector>
+
+#include "item.h"
+#include "debuff.h"
+
+class Player;
+
+class Potion: public Item {
```
Comment (alexgirard) [inline game/debuffHeal.h]: how come this class is named `Potion` when the file name is `debuffHeal`? Our UML shows both classes

File: game/potion.h [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+
+#include <string>
+#include <vector>
+
+#include "item.h"
+#include "statInfo.h"
+
+class Player;
+
+class Potion: public Item {
```
Comment (alexgirard) [inline game/potion.h:12]: Ya I think it was a copypasta error above

**Review by alexgirard — CHANGES_REQUESTED** [review]
Looks good! Small naming change

File: game/action.h [hunk 3]
```diff
@@ -0,0 +1,25 @@
+#ifndef ACTION_H
+#define ACTION_H
+
+#include <string>
+#include <vector>
+
+class Player;
+
+class Action {
+  virtual void doActionOverride(Player &player, const std::vector<size_t> &targets) const = 0;
```
Comment (ariellam) [inline game/action.h]: Remove param names from method declarations

File: game/debuffHeal.h [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+#include <string>
+#include <vector>
+
+#include "item.h"
+#include "debuff.h"
+
+class Player;
+
+class DebuffHeal: public Item {
+  DebuffName debuff;
```
Comment (ariellam) [inline game/debuffHeal.h]: can we make this const?

**Review by ariellam — APPROVED** [review]
lgtm 👍

**Review by alexgirard — APPROVED** [review]
Looks good!

**Pull request merged**